src/ansys/mapdl/core/cli/convert.py

```diff
@@ -63,7 +63,7 @@ def get_input_source(ctx, param, value):
 
         File mapdl.dat successfully converted to mapdl.out.""",
 )
-@click.argument("filename_in", callback=get_input_source, required=False)
+@click.argument("filename_in", callback=get_input_source, required=True)
```

Comment:
seems like you're mixing a change from another branch

Reply:
That is a bug I discovered while working on this. I didn't want to open an issue just for that. So I put it in here anyway. Probably a bad practice.